README.md

```diff
@@ -293,9 +293,9 @@ npm test -- mocha:<tag>:<suite>:[section]
 ```
 
 * Where **tag** can be one of `default | unstable | slow | extensive` (required)
-* Where **suite** can be one of `unit | functional | integration` (required)
+* Where **suite** can be one of `unit | functional | integration | network` (required)
```

Comment:
Probably this should reflect the layer order of the test.
ie: unit | integration| functional | network